internal/transport/transport.go

```diff
@@ -437,6 +444,12 @@ func (s *Stream) Status() *status.Status {
 	return s.status
 }
 
+// HeaderWireLength returns the size of theheaders of the stream as received
+// from the wire.
```

Comment:
.... "Valid only on the server"?

Reply:
Sure. Added. Does it not know/need to know the client side eventually?

Reply:
Maybe, but at that time we can fix the comment.

Honestly one of the next cleanups I'd love to do is to split the `transport.Stream` type into "client" and "server" versions instead of sharing code awkwardly where most of it needs to check a boolean to figure out whether it's a client or server stream and do radically different behavior depending on which.

Reply:
Hmmm ok sounds good.